### PR TITLE
Strike mentions of undefined term "automatic object"

### PR DESCRIPTION
**Pull request by sdkrystian**
"Automatic object" is not a term defined by the standard. Replacing all mentions of it with "object with automatic storage duration" resolves the issue.

### CODE REVIEW

**Comment by zygoloid** [comment]
This term is fairly widely used, and is shorter than the alternative; maybe we should just define it? [basic.stc.auto]p1:

> Block-scope variables not explicitly declared `static`, `thread_local`, or `extern` have <i>automatic storage duration</i><ins>; such objects are called <i>automatic objects</i></ins>.

**Comment by 18** [comment]
@zygoloid I feel that it's not used enough to warrant it. We already have a bunch of similar definitions like "local variable" and "local entity".

**Comment by jensmaurer** [comment]
I'd like to point out that @zygoloid's suggested change equates variables with objects, which we want to move away from. (There is a separate automatic object for each invocation of a function containing a non-static, non-thread-local, non-extern variable declaration.)

**Comment by jensmaurer** [comment]
Editorial meeting: use "with X storage duration", also cover static object.

File: source/classes.tex [hunk 1]
```diff
@@ -6555,7 +6555,7 @@
 \end{codeblock}
 Here the criteria for elision can
 eliminate
-the copying of the local automatic object
+the copying of the object with automatic storage duration
 \tcode{t}
```
Comment (zygoloid) [inline source/classes.tex]: This makes it sound like `t` is a name for the storage duration, not for the object.

Comment (sdkrystian) [reply source/classes.tex]: @zygoloid Changing it to "the object `t` with automatic storage duration" would resolve this

**Pull request merged**